recipes/pyre2/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pyre2" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```

Comment:
I believe that those quotes jinja variables will break the updating bot.

Reply:
Hmm, I took that directly from the docs (changing now).